src/preview/font_match.rs

```diff
@@ -152,21 +153,15 @@ mod with_fontconfig {
         pattern: *mut fc::FcPattern,
     ) -> Option<FontDescriptor> {
         let path = fc_pattern_get_string_property(pattern, FC_PROPERTY_FILE);
-        if path.is_none() {
-            return None;
-        }
+        path.as_ref()?;
         let path = path.unwrap();
```

Comment:
TIL: `?` can also be used to bail early on `Option::None` :) Thanks.

In that case I think lines like these:
```
let path = fc_pattern_get_string_property(pattern, FC_PROPERTY_FILE);
if path.is_none() {
    return None;
}
let path = path.unwrap();
```
should be simplified to:
```
let path = fc_pattern_get_string_property(pattern, FC_PROPERTY_FILE)?;
```

Reply:
It's all clippy :)

You're right -> d1c6d41